TrafficCapture/dockerSolution/src/main/docker/docker-compose.yml

```diff
@@ -81,8 +78,22 @@ services:
 #  traffic_comparator:
 #    image: ''
 
-#  opensearch-benchmarks:
-#    image: ''
+  opensearch-benchmarks:
+    image: 'migrations/open_search_benchmark:latest'
+    networks:
+      - migrations
+    command:
+      - /bin/sh
+      - -c
+      - >
+        echo "Running opensearch-benchmark w/ 'geonames' workload..." &&
+        opensearch-benchmark execute-test --distribution-version=1.0.0 --target-host=https://captureproxy:9200 --workload=geonames --pipeline=benchmark-only --test-mode --kill-running-processes --workload-params "target_throughput:0.5,bulk_size:10,bulk_indexing_clients:1,search_clients:1"  --client-options="use_ssl:true,verify_certs:false" &&
+        echo "Running opensearch-benchmark w/ 'http_logs' workload..." &&
+        opensearch-benchmark execute-test --distribution-version=1.0.0 --target-host=https://captureproxy:9200 --workload=http_logs --pipeline=benchmark-only --test-mode --kill-running-processes --workload-params "target_throughput:0.5,bulk_size:10,bulk_indexing_clients:1,search_clients:1"  --client-options="use_ssl:true,verify_certs:false" &&
+        echo "Running opensearch-benchmark w/ 'nested' workload..." &&
+        opensearch-benchmark execute-test --distribution-version=1.0.0 --target-host=https://captureproxy:9200 --workload=nested --pipeline=benchmark-only --test-mode --kill-running-processes --workload-params "target_throughput:0.5,bulk_size:10,bulk_indexing_clients:1,search_clients:1"  --client-options="use_ssl:true,verify_certs:false" &&
+        echo "Running opensearch-benchmark w/ 'nyc_taxis' workload..." &&
+        opensearch-benchmark execute-test --distribution-version=1.0.0 --target-host=https://captureproxy:9200 --workload=nyc_taxis --pipeline=benchmark-only --test-mode --kill-running-processes --workload-params "target_throughput:0.5,bulk_size:10,bulk_indexing_clients:1,search_clients:1"  --client-options="use_ssl:true,verify_certs:false"
```

Comment:
I'm interested in explorign using their pre-built image for this, but that's a task for another day.